### ISSUE TITLE
CQL-to-ELM Translator - Incorrect elm generated for ToString(argument Time)

### ISSUE THREAD

**Issue by sanumula** [cqframework/clinical_quality_language#454]
The cql-to-elm translator generates incorrect elm for the ToString function when Time is passed as an argument as shown below

CQL:
`define TimeOverload: ToString(@12:30:00.000)`

ELM:
```
<def name="TimeOverload" context="Patient" accessLevel="Public">
   <expression xsi:type="ToString">
      <signature name="t:Ratio" xsi:type="NamedTypeSpecifier"/>
      <operand xsi:type="Ratio">
         <numerator value="30" unit="1"/>
         <denominator value="0.000" unit="1"/>
      </operand>
   </expression>
</def>
```

**Comment by cmoesel** [comment]
Hi @sanumula.  In CQL, time literals start w/ `@T` -- so your code above should actually be:
```
define TimeOverload: ToString(@T12:30:00.000)
```
See: https://cql.hl7.org/02-authorsguide.html#date-datetime-and-time

Once you add the `T`, it should work as expected.  Did CQL-to-ELM report any errors regarding that line of code or did it run smoothly and just produce unexpected ELM?  I'd hope that it would have reported an error since `@12:30:00.000` isn't really a valid construct in CQL.

**Comment by sanumula** [comment]
@cmoesel Thanks for the catch! I ran the cql-to-elm translator with the valid Time construct and it is generating the elm properly as expected. I picked the ToString example from CQL spec[1] and it has the invalid format.

[1] https://cql.hl7.org/09-b-cqlreference.html#tostring

**Comment by p9g** [comment]
Thanks for catching the spec flaw.  STU comment submitted at http://www.hl7.org/dstucomments/showdetail_comment.cfm?commentid=1857

**Comment by cmoesel** [comment]
@brynrhodes -- you might want to look at how CQL-to-ELM handles this broken syntactical statement.  Even though the syntax was wrong, I would have expected CQL-to-ELM to balk rather than emit the ELM shown in the original report.

**Comment by brynrhodes** [comment]
Agreed, the translator should not be outputting the incorrect ELM there.

**Comment by brynrhodes** [comment]
Addressed in 1.4.9 release
